io.github.kovzol.bibref.yaml

```diff
@@ -0,0 +1,44 @@
+id: io.github.kovzol.bibref
+runtime: org.kde.Platform
+runtime-version: '6.5'
+sdk: org.kde.Sdk
+command: bibref-qt
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+modules:
+  - name: sword
+    buildsystem: cmake-ninja
+    builddir: true
+    sources:
+      - type: archive
+        url: https://www.crosswire.org/ftpmirror/pub/sword/source/v1.9/sword-1.9.0.tar.gz
+        sha256: 42409cf3de2faf1108523e2c5ac0745d21f9ed2a5c78ed878ee9dcc303426b8a
+  - name: boost
+    buildsystem: simple
+    build-commands: ['./bootstrap.sh --prefix=/app --with-libraries=system,filesystem', './b2 -j $FLATPAK_BUILDER_N_JOBS install']
+    sources:
+      - type: archive
+        url: https://boostorg.jfrog.io/artifactory/main/release/1.78.0/source/boost_1_78_0.tar.bz2
+        sha256: 8681f175d4bdb26c52222665793eef08490d7758529330f98d3b29dd0735bccc
+  - name: bibref-qt
+    buildsystem: cmake-ninja
+    subdir: qt
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+      - -DBUILD_SHARED_LIBS=ON
+      - -UPROJECT_SOURCE_DIR
+      - -DPROJECT_SOURCE_DIR=/app/share/bibref-qt
+    build-options:
+      env:
+        XDG_DATA_HOME: /app/home
+    sources:
+      - type: archive
+        url: https://github.com/kovzol/bibref/archive/85f8636bd3ff427c6678022017397bfb42785742.zip
+        sha256: f98542b0f91323627e77bb7839252056c3055c364742d12578a505335367e27d
+  - name: finalize
```

Comment:
no reason for this to be a separate module.

Reply:
I only need an extra command at the end, but if I put it in the previous module, it overwrites its build command.

Reply:
`post-install` is here for that.

Reply:
I see, thanks!